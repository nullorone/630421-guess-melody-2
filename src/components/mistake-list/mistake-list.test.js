import React from "react";
import renderer from 'react-test-renderer';
import MistakeList from "../mistake-list/mistake-list";

const Value = {
  EMPTY: 0,
  FULL: 1,
};

const initProps = {
  mistakes: Value.EMPTY,
};

it(`Snapshot render component`, () => {
  const tree = renderer
    .create(
        <MistakeList
          {...initProps}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
