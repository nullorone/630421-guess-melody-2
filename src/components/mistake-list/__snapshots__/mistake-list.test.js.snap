// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot render component 1`] = `null`;
